app/models/creative.rb

```diff
@@ -54,7 +55,14 @@ class Creative < ApplicationRecord
   scope :search_user_id, ->(value) { value.blank? ? all : where(user_id: value) }
   scope :standard, -> { where creative_type: ENUMS::CREATIVE_TYPES::STANDARD }
   scope :sponsor, -> { where creative_type: ENUMS::CREATIVE_TYPES::SPONSOR }
-
+  scope :order_by_status, -> {
+                            order_by = ["CASE"]
+                            ENUMS::CREATIVE_STATUSES.values.each_with_index do |status, index|
+                              order_by << "WHEN status='#{status}' THEN #{index}"
+                            end
+                            order_by << "END"
+                            order(Arel.sql(order_by.join(" ")))
+                          }
```

Comment:
I'd consider making this even more Arel driven. We can pair on that if you like.

Reply:
Yeah I'll need help with this.

Reply:
Lets just run with it for now. We can refactor later if desired.